content/plugin_community.md

```diff
@@ -27,63 +27,63 @@ The following Ohai plugins are available from the open source community:
 </thead>
 <tbody>
 <tr>
-<td><a href="https://github.com/demonccc/chef-ohai-plugins/blob/master/dell.rb">dell.rb</a></td>
+<td><a href="https://github.com/demonccc/chef-ohai-plugins/blob/main/dell.rb">dell.rb</a></td>
```

Comment:
I'd like you to confirm the name of the branches for the sites on this page. They link out to repos owned by community members, so we should check what the branch name is.

Reply:
Checked all the links and have updated the link accordingly.